website/docs/docs/on-premises-2x/environment-variables.md

```diff
@@ -643,4 +650,35 @@ Disables the new user interface introduced in version 2.0.0. When this environme
 
 **Required:** false<br/>
 **Default:** unset<br/>
-**Allowed values:** `true`<br/>
\ No newline at end of file
+**Allowed values:** `true`<br/>
+
+## Observability
+
+<hr/>
+
+
+
+**Allowed values:** `true`<br/>
+
+### OTEL_EXPORTER_OTLP_ENDPOINT
+
+The OTLP (HTTP) endpoint to export spans to.
+
+**Required**: false<br/>
+**Example**: `https://172.23.92.124:4318`<br/>
+**Allowed values:** `true`, `false`<br/>
```

Comment:
Do we support both `http` and `https`?

With `https`, do we support self-signed certificates?

Reply:
@pahnin please confirm.

We don't support self-signed to my knowledge, but perhaps it's possible to support it if it's in the keystore.

Reply:
If it's a self-signed certificate on the otel exporter service, the question is - how will PactFlow behave? Can we connect to a self-signed cert? 

Let's assume for now it can't, and if the question comes up, we can review it.